.clang-format

```diff
@@ -0,0 +1,10 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+UseTab: Never
+BreakBeforeBraces: Linux
+AlignAfterOpenBracket: Align
+AllowShortIfStatementsOnASingleLine: true
+AllowAllArgumentsOnNextLine: false
```

Comment:
I guess this should be turned to true for one of my requests?

Reply:
It's the BinPack arguments bellow I think. I'll play with it a bit more, I don't know all the options by heart. It's more like tune it once and then check if newer clang-format fixes something or adds new option that was missing.